xonsh/vox.py

```diff
@@ -0,0 +1,206 @@
+from os.path import join, basename, exists
+import os
+from shutil import rmtree
+import venv
+
+import builtins
+
+
+class Vox:
+    """Vox is a virtual environment manager for xonsh."""
+
+    def __init__(self):
+        """Ensure that $VIRTUALENV_HOME is defined and declare the available vox commands"""
+
+        if not builtins.__xonsh_env__.get('VIRTUALENV_HOME'):
+            if os.name == 'nt':
+                home_path = builtins.__xonsh_env__['USERPROFILE']
+
+            elif os.name == 'posix':
+                home_path = os.path.expanduser('~')
```

Comment:
Sorry, not just this line, but this whole conditional can be replaced with the expanduser().  Rather than having the if-elif-else blocks, you can just have the one line.